Sources/NavigationStack/NavigationStack.swift

```diff
@@ -5,10 +5,15 @@
 
 import SwiftUI
 
-/// The transition type for the whole NavigationStackView
+/// The transition type for the whole NavigationStackView.
 public enum NavigationTransition {
+    /// Transitions won't be animated.
     case none
+
+    /// A slide transition from the right to the left on push and a slide transition from the left to the right on pop.
     case `default`
+
+    /// To define a custom transition.
```

Comment:
```suggestion
    /// Use a custom transition.
```
It's not defined here. More over I'd personally avoid comments which just rephrase the field name. Those add no information

Reply:
The custom transition is defined from the caller of the API. About the documentation not saying anything more you're right. Actually I can (and I should) say a little bit more. I'll fix this.

Reply:
Fixed.